pkgs/applications/blockchains/stellar-core.nix

```
@@ -1,31 +1,26 @@
-{ lib, stdenv, fetchgit, autoconf, libtool, automake, pkg-config, git
+{ lib, stdenv, fetchFromGitHub, autoconf, libtool, automake, pkg-config, git
 , bison, flex, postgresql }:
 
-let
+stdenv.mkDerivation rec {
   pname = "stellar-core";
-  version = "0.5.1";
+  version = "17.0.0";
 
-in stdenv.mkDerivation {
-  name = "${pname}-${version}";
-
-  src = fetchgit {
-    url = "https://github.com/stellar/stellar-core.git";
-    rev = "refs/tags/v${version}";
-    sha256 = "0ldw3qr0sajgam38z2w2iym0214ial6iahbzx3b965cw92n8n88z";
+  src = fetchFromGitHub {
+    owner = "stellar";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1ngl8yjqb8xzhdwzlxzzxf14q2hgwy2ysb17sn5380rrn0jswin1";
     fetchSubmodules = true;
-    leaveDotGit = true;
   };
 
-  nativeBuildInputs = [ pkg-config ];
-  buildInputs = [ autoconf automake libtool git ];
+  nativeBuildInputs = [ automake autoconf git libtool pkg-config ];
```

```suggestion
  nativeBuildInputs = [ autoreconfHook pkg-config ];
```
and drop `preConfigure`

---

Fails to build:

```
automake: error: cannot open < lib/medida.mk: No such file or directory
autoreconf: error: automake failed with exit status: 1
builder for '/nix/store/gmd8jwvgdp8x2fh554978960gmr6mdxi-stellar-core-17.0.0.drv' failed with exit code 1
error: build of '/nix/store/gmd8jwvgdp8x2fh554978960gmr6mdxi-stellar-core-17.0.0.drv' failed
```